docs/install/prerequisites.md

```diff
@@ -1,11 +1,18 @@
 # Prerequisites
 
+Portable (CO:RE) option:
 - Linux kernel version >= 4.18
-- Linux kernel headers available under conventional location (see [Linux Headers](../headers) section for more info)
+- `CONFIG_DEBUG_INFO_BTF` Linux compile option enabled (only needed for running, not compilation)
 - libc, and the libraries: libelf, zlib
 - GNU Make >= 4.3
-- clang >= 9
+- clang >= 11
```

Comment:
I think with CO:RE we can start to remove the compile-time requirements from here, since this is the first thngs the user sees about tracee, and it's quite intimidating. wdyt about moving this to the ebpf-compilation doc?